### PR TITLE
Learning Howto - Reverse Proxy + custom URL + GSSAPI in war config

### PR DESCRIPTION
**Pull request by AndLLA**
Hallo,
I created a new pull request, as the previous one was closed.
https://github.com/rundeck/docs/pull/1061

I hope I followed the suggestion in the correct way.

Thanks

### CODE REVIEW

File: docs/.vuepress/sidebar-menus/learning.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ module.exports = [{
               '/learning/howto/migrate-to-mysql.md',
               '/learning/howto/use-terraform-provider.md',
               '/learning/howto/learn-rd-cli.md',
-              '/learning/howto/acl_basic_examples.md'
+              '/learning/howto/acl_basic_examples.md',
+              '/learning/howto/apache2-proxy-gssapi.md',
```
Comment (fdevans) [inline docs/.vuepress/sidebar-menus/learning.js]: remove trailing comma or the menus may not build successfully

File: docs/learning/howto/apache2-proxy-gssapi.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+which is an easy way to implement SSO where Active Directory (or Kerberos) is available.
+
+
+
+# Prerequisites
+
+We assume that inside the VM:
+* Rundeck has been already set up with all the necessary configuration options for DB, logging, etc.
+* apache2 is running with the necessary modules enabled (proxy, gssapi, etc)
+* a valid keytab has been generated and contains the correct principal keys (in this case HTTP/srv.example.com@EXAMPLE.COM)
```
Comment (fdevans) [inline docs/learning/howto/apache2-proxy-gssapi.md]: Capital A

File: docs/learning/howto/apache2-proxy-gssapi.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+The `GSSAPI` (and related `RequestHeader`) configuration is of course optional 
+and if applied should be customized with the actual location of the keytab files and domain name `EXAMPLE.COM`.
+In addition the stripping of the domain name `EXAMPLE.COM` from the `Uiid` field should be updated as well, if needed.
+
+Last but not least, the look up of user roles, necessary for the correct execution of the authorization process on Rundeck side, 
+is implemented by reading one file named as the user principal and located under the folder `/etc/apache2/rundeck-roles/`.
+The file contains one row with the list of roles specific to the user, for example `admin,user`.
+
+This approach is very simple and perhaps suitable only for a small user base
```
Comment (fdevans) [inline docs/learning/howto/apache2-proxy-gssapi.md]: This comment should likely go up at the top in the description of the article/solution.

**Review by fdevans — CHANGES_REQUESTED** [review]
Had to make comments on minor changes that I'd normally handle myself, but GitHub won't let me edit your fork PR.  If you can make these small updates I'll pass it to the technical team to validate.

File: docs/.vuepress/sidebar-menus/learning.js (hunk 1 above)
Comment (fdevans) [reply docs/.vuepress/sidebar-menus/learning.js]: Did you happen to test the build?

**Comment by AndLLA** [comment]
Hallo,
I fixed the typos and moved the comment to the introduction.
I tested the build (previously, and after the fixes) and it seems to work (note that I'm using the latest vuepress,
because some of the pinned dependencies seem to require python2, which I do not have anymore).

Thanks

rundeck_docs_dev_run

**Comment by fdevans** [comment]
Great, thank you.  @MegaDrive68k will help test the techincal bits and let us know his feedback soon.

**Review by MegaDrive68k — APPROVED** [review]
It works in my environment, all good :) Thank you @AndLLA

**Review by fdevans — APPROVED** [review]

**Comment by fdevans** [comment]
@AndLLA If you can resolve the last bit of conflicts I can hopefully merge before our 4.6.0 release.

**Comment by AndLLA** [comment]
Hallo,
the conflict should be fixed.

Thanks !!!

**Review by fdevans — APPROVED** [review]

**Pull request merged**